payment/migrations/0002_auto_20170615_1426.py

```diff
@@ -6,8 +6,825 @@
 from django.db import migrations, models
 from payment.models import TermsAndConditions, TranslatedTerms, PAYMENT_TERMS
 
+LICENCE_EN = """
+General Terms and Conditions of Use
```

Comment:
Not sure how you end up adding this migration as `payment` application does not belong to whitbrand environments and is not in `INSTALLED_APPS`. Anyway I think this version of FUN's terms and conditions do not apply to white brands.

Reply:
Good point! For now I've applied `dogwood.3-fun` settings in the `eucalyptus.3-wb` build (WIP). I had doubts about the English version but I have to confess that I didn't paid that much attention to it, sorry for that.

Anyway, if you don't mind I'll switch to an empty string as this migration is half broken.

Reply:
👍